src/assets/css/themes/darkly-compact.scss

```diff
@@ -55,5 +56,3 @@ hr.my-3 {
     display: none;
   }
 }
-
-@import "../../../../node_modules/bootstrap/scss/bootstrap";
```

Comment:
I think it's necessary that this come at the end, since `$container-max-widths` is being written here.

Also, I believe as of Bootstrap 5, `$container-max-widths` (and other maps) need to be spelled out in full; they're not longer merged with the defaults.

Reply:
> I think it's necessary that this come at the end, since $container-max-widths is being written here.

If I put the bootstrap import at the end, wouldn't that override the variables set higher up in the file?

Reply:
No, Bootstrap uses the Sass `!default` flag so as not to overwrite any variables you've set.

Reply:
No, Bootstrap uses Sass's `!default` flag so as not to overwrite any vars you may have set yourself